Apeswap-Bots/Large-LP-Deposits-Withdrawals/README.md

```diff
@@ -0,0 +1,44 @@
+# Large Apeswap LP Deposit/Withdrawal Bot
+
+## Description
+
+This bot detects LP deposits and withdrawals in Apeswap pools when the `amount0`/`amount1` and the pool's `totalSupply` are high.
+> The thresholds `AMOUNT_THRESHOLD_PERCENTAGE`, `POOL_SUPPLY_THRESHOLD` of what is considered high, can be adjusted in **src/utils.ts**.
+
+> To switch to Polygon network, set `onPolygon` to `true` in **src/utils.ts**.
```

Comment:
Let's avoid this. Use the `initialize` function to detect the network and select the addresses accordingly.